### PR TITLE
Add header

### PR DESCRIPTION
**Pull request by spriha1301**
header

### CODE REVIEW

File: blogs/01/styles/header.css [hunk 1]
```diff
@@ -2,7 +2,7 @@ nav.header{
     background-color:white;
```
Comment (itsashishbharti) [inline blogs/01/styles/header.css:2]: use variable  for  background color

**Review by itsashishbharti — COMMENTED** [review]
pull the all yours entire html and css code of  header section

**Comment by itsashishbharti** [comment]
share your mobile screenshot of header section

File: blogs/01/index.html [hunk 2]
```diff
@@ -18,12 +18,12 @@ <h3>Mini Blog</h3>
...
                     <ul class="menu">
-                        <li><a href="#">Home</a></li>
-                        <li><a href="#">Politics</a></li>
-                        <li><a href="#">Tech</a></li>
-                        <li><a href="#">Entertainment</a></li>
-                        <li><a href="#">Travel</a></li>
-                        <li><a href="#">Sports</a></li>
+                        <li><a href="#">home</a></li>
+                        <li><a href="#">politics</a></li>
+                        <li><a href="#">tech</a></li>
```
Comment (amezng) [inline blogs/01/index.html:23]: Use `#home` as the href value. Similarly for all the other anchor elements

**Review by amezng — CHANGES_REQUESTED** [review]
- [ ] Upload the mobile view
- [ ] Use css variables

**Comment by spriha1301** [comment]
mobile view

File: blogs/01/styles/post-highlight-section-1.css [hunk 3]
```diff
@@ -30,7 +30,7 @@ div.highlight-section-wrapper>div.card-section-1>div.grid-box{
     div.highlight-section-wrapper>div.card-section-1>div.grid-box-1{
         margin-top: 35px;
     }
-    div.voucher{
+    div.boxes{
```
Comment (itsashishbharti) [inline blogs/01/styles/post-highlight-section-1.css:33]: In header section html you not deleare any div with the class name .boxes why you use this

**Comment by amezng** [comment]
not as per the design

**Pull request closed without merging**